packages/admin/src/middleware.page.ts

```diff
@@ -42,6 +43,6 @@ export function middleware(req: NextRequest, ev: NextFetchEvent) {
 
     return res;
   } else {
-    return NextResponse.redirect(process.env.LOGIN_URL!);
+    return NextResponse.redirect(getLoginUrl()!);
```

Comment:
if we move the non-null assertions inside `getLoginUrl` we don't need to add them everywhere it's called like this.

Reply:
```js
const getLoginUrl = () => 
  process.env.ONE_LOGIN_ENABLED ? process.env.V2_LOGIN_URL! : process.env.LOGIN_URL!;

```

Reply:
> We can avoid non-null assertions here too if we provide a definition `environment.d.ts`.
> 
> ```
> declare global {
>   namespace NodeJS {
>     interface ProcessEnv {
>       //...
>       ONE_LOGIN_ENABLED: string
>     }
>   }
> }
> 
> export { }
> ```

Would this be better solved in a ticket to remove all non-null assertions?

Reply:
I think non-null assertions in general are fine, but we can avoid the need here specifically.

Reply:
have added those types